camera_topic: /head_xtion/rgb/image_raw
depth_camera_topic: /head_xtion/depth/image_raw
yolo_topic: /darknet_ros/bounding_boxes
people_topic: /people_tracker_filter
legs_topic: /leg_tracker_measurements
entities_marker_topic: /entities_marker
leg_tolerance: 0.6
camera_tolerance: 1.0
people_tolerance: 2.0
entity_decay: 0.01
entity_cumulation: 0.2
entity_friction: 0.8
name_weight: 2.0
person_weight: 2.0
color_weight: 0.2
gender_weight: 0.6
publication_threshold: 0.2
max_probability: 1
leg_probability_ratio: 0.01
speed_ratio: 8

# Entity kinematic parameters
entity_decay: 0.02
entity_friction: 0.75
publication_threshold: 0.2
max_probability: 1
speed_ratio: 10

# post merge parameters
post_merge_tolerence: 1.0
post_merge_max_distance: 0.2
post_merge_person_tolerence_ratio: 0.8
post_merge_person_max_distance: 0.8

# leg parameters
leg_merge_tolerance: 1.0
leg_merge_max_distance: 0.8
leg_merge_cumulation: 0.005

# Camera parameters
camera_merge_tolerance: 1.4
camera_merge_max_distance: 0.4
camera_merge_cumulation: 0.9
people_merge_tolerance: 1.5
people_merge_max_distance: 0.8
people_merge_cumulation: 0.9

# Weights parameters
name_weight: 1
color_weight: 0.2
gender_weight: 0.5